code/modules/clothing/spacesuits/rig/suits/light.dm

```diff
@@ -62,10 +62,12 @@
 
 /obj/item/clothing/suit/lightrig/hacker
 	siemens_coefficient = 0.4
+	var/holder = null //holder is required for suits
 
 /obj/item/clothing/shoes/lightrig/hacker
 	siemens_coefficient = 0.4
 	flags = NOSLIP //All the other rigs have magboots anyways, hopefully gives the hacker suit something more going for it.
+	var/magboot_state = null //magboot_state must be defined for all shoes
```

Comment:
With the above two vars, what's being aimed for? That is to say, you've created a holder var and a magboot_state var, but they're not used.